examples/ownership-transfer/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "ownership-transfer",
+  "version": "1.0.0",
+  "description": "An example of ownership transfer with Tupelo",
+  "main": "transfer.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
maybe a "run" here?

Reply:
@tobowers I don't think a bare `run` will work, I added `start` instead, which might be what you want?